metricbeat/module/nats/connection/_meta/fields.yml

```diff
@@ -4,10 +4,46 @@
     Contains nats connection related metrics
   release: ga
   fields:
+    - name: id
+      type: keyword
+      description: >
+        The id of the connection
```

Comment:
```suggestion
        The ID of the connection
```
or simply 
```suggestion
        Connection ID
```

Reply:
I'll correct this manually as I likely need to update the markdown docs as well.

Reply:
Casing should now be corrected in `fields.yml` as well as the Markdown docs.